common/src/test/java/com/thoughtworks/go/domain/materials/git/GitCommandTest.java

```diff
@@ -268,6 +268,22 @@ public void shouldRetrieveLatestModification() throws Exception {
         assertThat(files.get(0).getAction(), Matchers.is(ModifiedAction.modified));
     }
 
+    @Test
+    public void shouldRetrieveLatestModificationWhenColoringIsSetToAlways() throws Exception {
+        setColoring();
+        Modification mod = git.latestModification().get(0);
+        assertThat(mod.getUserName(), is("Chris Turner <cturner@thoughtworks.com>"));
+        assertThat(mod.getComment(), is("Added 'run-till-file-exists' ant target"));
+        assertThat(mod.getModifiedTime(), is(parseRFC822("Fri, 12 Feb 2010 16:12:04 -0800")));
+        assertThat(mod.getRevision(), is("5def073a425dfe239aabd4bf8039ffe3b0e8856b"));
+
+        List<ModifiedFile> files = mod.getModifiedFiles();
+        assertThat(files.size(), is(1));
+        assertThat(files.get(0).getFileName(), is("build.xml"));
+        assertThat(files.get(0).getAction(), Matchers.is(ModifiedAction.modified));
+        unsetColoring();
```

Comment:
If the test fails, `unsetColoring` would not be called. Could presumably affect other tests.